libdnf/rpm/solv/solv_map.hpp

```diff
@@ -150,24 +152,42 @@ inline SolvMap::SolvMap(int size) {
 }
 
 
-inline SolvMap::SolvMap(const Map * other) {
-    map_init_clone(&map, other);
+inline SolvMap::SolvMap(const Map * map) {
+    map_init_clone(&this->map, map);
 }
 
-
 inline SolvMap::SolvMap(const SolvMap & other) : SolvMap(other.get_map()) {}
 
 inline SolvMap::SolvMap(SolvMap && other) noexcept {
-    map.size = other.map.size;
```

Comment:
I suggest that you have to copy a map size `map.size = other.map.size;` for a correct functionality.